vnet/lib/vnet/openflow/ovs_ofctl.rb

```diff
@@ -37,20 +37,36 @@ def add_flow(flow)
       debug log_format("'#{command}' => #{result}") if verbose
     end
 
+    # TODO: Deprecate.
     def add_ovs_flow(flow_str)
       command = "#{@ovs_ofctl} add-flow #{switch_name} #{flow_str}"
       result = system(command)
 
       debug log_format("'#{command}' => #{result}") if verbose
     end
 
+    # TODO: Deprecate.
     def add_ovs_10_flow(flow_str)
       command = "#{@ovs_ofctl_10} add-flow #{switch_name} #{flow_str}"
       result = system(command)
 
       debug log_format("'#{command}' => #{result}") if verbose
     end
 
+    def add_ovs_flow2(flow_str)
```

Comment:
I assume a code-wide search and replace will be done to rename this to `add_ovs_flow` once all the usage of the former method has been eliminated?